articles/migrate/concepts-azure-vmware-solution-assessment-calculation.md

```diff
@@ -285,9 +285,9 @@ As the pricing for Azure VMware Solution (AVS) is per node, the total cost does
 
 In the Azure readiness report for Azure VMware Solution (AVS) assessment, you can see the following suggested tools:
 
-- **VMware HCX or Enterprise**: For VMware servers, VMware Hybrid Cloud Extension (HCX) solution is the suggested migration tool to migrate your on-premises workload to your Azure VMware Solution (AVS) private cloud. [Learn More](../azure-vmware/install-vmware-hcx.md).
-- **Unknown**: For servers imported via a CSV file, the default migration tool is unknown. Though for VMware servers, it is recommended to use the VMware Hybrid Cloud Extension (HCX) solution.
+- **VMware HCX or Enterprise**: For VMware vSphere servers, VMware Hybrid Cloud Extension (HCX) solution is the suggested migration tool to migrate your on-premises vSphere workload to your Azure VMware Solution (AVS) private cloud. [Learn More](../azure-vmware/install-vmware-hcx.md).
+- **Unknown**: For servers imported via a CSV file, the default migration tool is unknown. Though for VMware vSphere servers, it is recommended to use the VMware Hybrid Cloud Extension (HCX) solution.
 
 ## Next steps
 
-Create an assessment for [AVS VMware VMs](how-to-create-azure-vmware-solution-assessment.md).
+Create an assessment for [AVS VMware Solution VMs](how-to-create-azure-vmware-solution-assessment.md).
```

Comment:
@rvandenbedem we can't use 'AVS' in published docs, must be spelled out 'Azure VMware Solution'. Please make this correction, thank you.

Reply:
@suzizuber - done.